server.py

```diff
@@ -6,3 +6,8 @@
 @server.get("/hello")
 def hello():
     return {"message": "hello"}
+
+
+@server.get("/wassup")
+def wassup():
+    return {"message": "wassup"}
```

Comment:
add exclamation mark
```suggestion
    return {"message": "wassup!"}
```